src/lib/main.js

```diff
@@ -16,16 +18,16 @@ XPCOMUtils.defineLazyModuleGetter(this, 'console',
 XPCOMUtils.defineLazyModuleGetter(this, 'CustomizableUI',
   'resource:///modules/CustomizableUI.jsm');
 
-var EXPORTED_SYMBOLS = ['Main']; // eslint-disable-line no-unused-vars
+const EXPORTED_SYMBOLS = ['Main']; // eslint-disable-line no-unused-vars
 
-var onTabSelect = function() { console.log('onTabSelect'); };
-var onTabOpen = function() { console.log('onTabOpen'); };
-var onTabClose = function() { console.log('onTabClose'); };
+const onTabSelect = function() { console.log('onTabSelect'); };
```

Comment:
Turns out that these are the two new ESLint rules we'd need:
```yaml
  func-names: 1
  func-style: [2, declaration]
```

I'm not 100% sold on the `func-names` rule. It means we'd need to give EVERY anonymous function a name (so all our Gulp tasks, everything in the inert test/test-index.js file, the ~9 anon functions in src/lib/util/Broker.js, etc).
We can set the rule to `0` (ignore) or `1` (warning), or just not include it if we don't really care.

And since this really isn't the place to have this discussion, I'll file a new bug.